protostar-cli/src/lib.rs

```diff
@@ -0,0 +1,82 @@
+use anyhow::{anyhow, Result};
+use camino::Utf8PathBuf;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use starknet::{
+    accounts::SingleOwnerAccount,
+    core::{chain_id, types::FieldElement},
+    providers::jsonrpc::{HttpTransport, JsonRpcClient},
+    signers::{LocalWallet, SigningKey},
+};
+use std::fs;
+use url::Url;
+
+#[derive(Deserialize, Serialize)]
+struct ParsedAccount {
+    private_key: String,
+    public_key: String,
+    address: String,
+}
```

Comment:
```suggestion
struct Account {
    private_key: String,
    public_key: String,
    address: String,
}
```
probably a better name because it being represented as a structure implies it being parsed